.github/workflows/tox.yml

```diff
@@ -37,29 +37,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [3.8,"3.10"]
-        django: [32,40,41,main]
+        python: [3.8,"3.10", "3.11"]
+        django: [32,41,main]
 
     env:
       TOXENV: py${{ matrix.python }}-django${{ matrix.django }}
 
-    services:
-      postgres:
-        image: postgres:12
-        env:
-          POSTGRES_USER: postgres
-          POSTGRES_PASSWORD: postgres
-          POSTGRES_DB: test
-          # Set health checks to wait until postgres has started
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-          # Maps tcp port 5432 on service container to the host
-          - 5432:5432
-
```

Comment:
Postgres is no longer required in CI as we are dropping Django 3.0, 3.1 which only support a Postgres JSON field type. 3.2+ support `JSONField` across all backends, including SQLite, so we run against that.